### DISCUSSION TITLE
after update 1.2.2 version , do not work c/c++ extention

### DISCUSSION THREAD

**Discussion by Denia-park** [microsoft/vscode-cpptools#7121] [Q&A]
Hi

I have a problem after the version update.
After updating to version 1.2.2, the extension program does not work even if i open the c file or h file.

When I rolled back to version 1.2.1, the extension program worked fine.

what is the problem??

**Comment by Denia-park** [comment]
After turning off and turning on the computer, I updated it from version 1.2.1 to 1.2.2, and suddenly the extension program works.

I don't know why.

**Reply by sean-mcmanus** [reply]
We suspect there's some VS Code installer bug -- we need to find some way repro it to report the bug to the VS Code team.

**Comment by rambo552** [comment]
Hi,
I am using Mac with M1 chip, when install this Extension, I face the same issue.
I tried to downgrade the version to last the where arm64 is supported,
however the issue remains same.

when fall back to intel chip version, it works fine.

any suggestions on how to get over this please.

Thanks and Regards.

**Reply by sean-mcmanus** [reply]
@michelleangela Any ideas?
